docs/_docs/user-guide/eldritch.md

```diff
@@ -635,6 +636,36 @@ The <b>process.netstat</b> method returns all information on TCP, UDP, and Unix
 
 ---
 
+## Report
+
+The report library is designed to enabled reporting structured data to Tavern. It's API is still in the active development phase, so **future versions of Eldritch may break tomes that rely on this API**.
+
+### report.file
+
+`report.file(path: str) -> None`
+
+Reports a file from the host that an Eldritch Tome is being evaluated on (e.g. a compromised system) to Tavern. It has a 1GB size limit, and will report the file in 1MB chunks. This process happens asynchronously, so after `report.file()` returns **there are no guarantees about when this file will be reported**. This means that if you delete the file immediately after reporting it, it may not be reported at all (race condition).
+
+### report.process_list
+
+`report.process_list(list: List<Dict>) -> None`
```

Comment:
I believe it returns a List<Dict> right?

Reply:
No